docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -177,6 +179,7 @@ spec:
     port: 80
   subroutes:
   - path: /coffee/latte
+    dos: default/dos-protected
```

Comment:
this comment is similar to https://github.com/nginxinc/kubernetes-ingress/pull/2241#discussion_r768093005  because dos is not the primary use case and it is also a paid addon, I don't think we should it include in in the main YAML example in this doc. I think the added documentation and the GitHub examples cover it enough.

Reply:
apologies, I missed the VSR. is it okay to leave the field in the table at least ?

Reply:
yep. thanks!